virttest/utils_test/__init__.py

```diff
@@ -447,70 +447,62 @@ def run_file_transfer(test, params, env):
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
     login_timeout = int(params.get("login_timeout", 360))
-
     error_context.context("Login to guest", logging.info)
-    session = vm.wait_for_login(timeout=login_timeout)
```

Comment:
I think "session = vm.wait_for_login(timeout=login_timeout)" should be here, can not move to line 494. Or it will cause case failed: No ipv4 DHCP lease for MAC 9a:e0:e1:e2:e3:e4, Client said 'connection refused'

if "session = vm.wait_for_login(timeout=login_timeout)" is here, the test will be passed.
(1/3) smp_16.32768m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.file_transfer: PASS (288.37 s)
 (2/3) smp_16.32768m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.file_transfer.vhost_force: PASS (158.17 s)
 (3/3) smp_16.32768m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.file_transfer.vhost_force.vhostforce_on.force_disable_msi: PASS (220.78 s)